src/features/marketplace/process/commands/list-item/list-item-command.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers } from 'ethers';
+import { IMarketplaceService } from '../../../services/marketplace-service.interface';
+import MarketplaceError from '../../enums/marketplace-error.enum';
+import ICommand from '../../interfaces/command.interface';
+import MarketplaceState from '../../types/marketplace-state';
+
+export default class MakeItemNftCommand implements ICommand<MarketplaceState> {
```

Comment:
You should put a more meaningful name here, since the command is meant to list items, and not creating new ones.